docs/platform/README.md

```diff
@@ -33,29 +33,50 @@ $ serverless -v
 
 ## Usage
```

Comment:
@eahefnawy Can you please remove everything except the login, logout and emit command and simply point users to the main serverless platform docs